backend/clubs/models.py

```diff
@@ -698,6 +698,37 @@ def send_approval_email(self, request=None, change=False):
                 reply_to=settings.OSA_EMAILS + [settings.BRANDING_SITE_EMAIL],
             )
 
+    def get_latest_and_latest_approved_versions(self):
+        """
+        Returns the latest and latest approved versions of club key fields
+        [name, description, image] contents that trigger reapproval.
+        """
+        latest_approved_version = (
+            self.history.filter(approved=True).order_by("-history_date").first()
+        )
+        latest_version = self.history.order_by("-history_date").first()
+
+        # if this is the first time the club is being approved
+        if not latest_approved_version:
+            return {
+                self.code: {
```

Comment:
would prefer to add the club code to the response in the view instead of this helper

Reply:
I think if we make the route instance-specific, we won't require the club code. With this new implementation, it looks like the approval queue list will remain as is, only that the links to a club in the queue are to `club\code\diff`

Reply:
I don't think we'll point to a new page for clubs in the queue (e.g. right now clicking on a club in the queue goes to `/api/clubs/<code>`). 

Instead, the idea is to expose an endpoint `/diff` that returns some historical comparison (either the last approved version, or the diffs themselves). We'll then add a param when an admin clicks on a club in the queue (e.g. `/api/clubs/<code>?review=True`) that triggers some frontend logic. The frontend logic is what's going to call `/api/clubs/<code>/diff` and display the results in some fashion. 

@julianweng @rm03 let me know if that's what we were thinking here.

Reply:
I see, that makes sense. I was actually thinking that there would be code repetition (with `/api/clubs/<code>` route) if implemented the way I described it earlier